client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -201,17 +217,21 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 		}
 	}
 
-	function previewDesign( _selectedDesign: Design ) {
-		recordTracksEvent(
-			'calypso_signup_design_preview_select',
-			getEventPropsByDesign( _selectedDesign )
-		);
+	function previewDesign( _selectedDesign: Design, positionIndex?: number ) {
+		recordTracksEvent( 'calypso_signup_design_preview_select', {
+			...getEventPropsByDesign( _selectedDesign ),
+			...( positionIndex && { position_index: positionIndex } ),
+		} );
 
 		setSelectedDesign( _selectedDesign );
 		setIsPreviewingDesign( true );
 	}
 
 	function viewMoreDesigns() {
+		recordTracksEvent( 'calypso_signup_design_view_more_select', {
+			flow: showGeneratedDesigns ? 'generated-design-step' : flow,
```

Comment:
I think this would currently only show `generated-design-step` right? Because it's only shown in the generated design picker

Reply:
Yes, wanted to keep the function more for general use, but I suppose we don't need it for now 🙂